strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-nuget-layout-provider/src/test/java/org/carlspring/strongbox/storage/metadata/nuget/TempNupkgFileTest.java

```diff
@@ -1,42 +1,23 @@
-/*
- * Copyright 2019 Carlspring Consulting & Development Ltd.
- * Copyright 2014 Dmitry Sviridov
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
```

Comment:
@ptirador : Was there a reason to remove these license headers?

Earlier this year, (I believe around April/May 2019), we actually absorbed the [JNuGet](https://bitbucket.org/aristar/jnuget/wiki/Home) project. The original author was happy for us to take over and continue developing it and we promised to give them credit via their icense notice.

When we absorbed their project, we actually inherited their test cases as well, which is why some of them will have these license.

Would you mind restoring these and anywhere else you may have removed them from (plese, double-check the pull request for other places)...? Thanks! :)

Reply:
Sorry, my bad. Thanks for the clarification, now it has been restored.

Reply:
No problem! Thanks for the quick fix! :)